utils/swift_build_support/swift_build_support/products/cmake_product.py

```diff
@@ -132,3 +134,645 @@ def install_with_cmake(self, install_targets, install_destdir):
+
+            # swift_cmake_options+=(
+            #     -DSWIFT_DARWIN_DEPLOYMENT_VERSION_OSX="${DARWIN_DEPLOYMENT_VERSION_OSX}"
+            #     -DSWIFT_DARWIN_DEPLOYMENT_VERSION_IOS="${DARWIN_DEPLOYMENT_VERSION_IOS}"
+            #     -DSWIFT_DARWIN_DEPLOYMENT_VERSION_TVOS="${DARWIN_DEPLOYMENT_VERSION_TVOS}"
+            #     -DSWIFT_DARWIN_DEPLOYMENT_VERSION_WATCHOS="${DARWIN_DEPLOYMENT_VERSION_WATCHOS}"
+            #     -DCMAKE_OSX_SYSROOT:PATH="${cmake_os_sysroot}"
+            #     # This is needed to make sure to avoid using the wrong architecture
+            #     # in the compiler checks CMake performs
+            #     -DCMAKE_OSX_ARCHITECTURES="${architecture}"
+            # )
+
+            swift_cmake_options.define('SWIFT_DARWIN_DEPLOYMENT_VERSION_OSX',
+                                       self.args.darwin_deployment_version_osx)
+            swift_cmake_options.define('SWIFT_DARWIN_DEPLOYMENT_VERSION_IOS',
+                                       self.args.darwin_deployment_version_ios)
+            swift_cmake_options.define('SWIFT_DARWIN_DEPLOYMENT_VERSION_TVOS',
+                                       self.args.darwin_deployment_version_tvos)
+            swift_cmake_options.define('SWIFT_DARWIN_DEPLOYMENT_VERSION_WATCHOS',
+                                       self.args.darwin_deployment_version_watchos)
+            swift_cmake_options.define('CMAKE_OSX_SYSROOT:PATH', cmake_os_sysroot)
+            swift_cmake_options.define('CMAKE_OSX_ARCHITECTURES', arch)
+
+            # lldb_cmake_options+=(
+            #     -DCMAKE_OSX_DEPLOYMENT_TARGET:STRING="${cmake_osx_deployment_target}"
+            #     -DCMAKE_OSX_SYSROOT:PATH="${cmake_os_sysroot}"
+            #     -DCMAKE_OSX_ARCHITECTURES="${architecture}"
+            # )
+            # llbuild_cmake_options+=(
+            #     -DCMAKE_OSX_ARCHITECTURES="${architecture}"
+            # )
+            # ;;
+
+        # # We don't currently support building compiler-rt for cross-compile targets.
+        # # It's not clear that's useful anyway.
+        # if [[ $(is_cross_tools_host "${host}") ]] ; then
+        #   llvm_cmake_options+=(
+        #       -DLLVM_TOOL_COMPILER_RT_BUILD:BOOL=FALSE
+        #       -DLLVM_BUILD_EXTERNAL_COMPILER_RT:BOOL=FALSE
+        #   )
+        # else
+        #   llvm_cmake_options+=(
+        #       -DLLVM_TOOL_COMPILER_RT_BUILD:BOOL="$(
+        #                         false_true ${SKIP_BUILD_COMPILER_RT})"
+        #       -DLLVM_BUILD_EXTERNAL_COMPILER_RT:BOOL="$(
+        #                         false_true ${SKIP_BUILD_COMPILER_RT})"
+        #   )
+        # fi
+
+        if self.is_cross_compile_target(host_target):
+            llvm_cmake_options.define('LLVM_TOOL_COMPILER_RT_BUILD:BOOL', 'FALSE')
+            llvm_cmake_options.define('LLVM_BUILD_EXTERNAL_COMPILER_RT:BOOL', 'FALSE')
+        else:
+            # TODO: is build-script-impl wrong here and these should be negated?
```

Comment:
I think the issue is that `build-script-impl` is negating `SKIP_BUILD_COMPILER_RT` whereas you're using your new `args.build_compiler_rt`, which I presume is defined as the inverse.